.github/workflows/coprocessor-cargo-tests.yml

```diff
@@ -95,3 +95,8 @@ jobs:
       run: |
         DATABASE_URL=postgresql://postgres:postgres@localhost:5432/coprocessor TXN_SENDER_TEST_GLOBAL_LOCALSTACK=1 cargo test --release -- --test-threads=1
       working-directory: coprocessor/fhevm-engine
+
+    - name: Run tests with incidents
+      run: |
+        DATABASE_URL=postgresql://postgres:postgres@localhost:5432/coprocessor TXN_SENDER_TEST_GLOBAL_LOCALSTACK=1 cargo test -p fhevm-listener --release --features test-simulate-reorg -- test_listener_restart
```

Comment:
As discussed, we should use a test db instead of relying on global ENV in CI.

Reply:
This is on release branch.